src/main/java/frc/robot/vision/GyroAngleValues.java

```diff
@@ -0,0 +1,9 @@
+package frc.robot.vision;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+/**
+ * A record that represents the gyro's angle which is passed into the limelight's network tables, to the robot_orientation_set entry. Its values
+ * were found in limelightlib sourcecode. The limelight is using it to calculate the BotPose2 position.
+ */
+public record GyroAngleValues(Rotation2d yaw, double yawRate, Rotation2d pitch, double pitchRate, Rotation2d roll, double rollRate) {}
```

Comment:
consider using Rotation3d for the gyro angle

Reply:
see [this comment](https://github.com/GreenBlitz/GB-Robot-Template/pull/470#discussion_r1871400965)

Reply:
I don't see why you can't just write `gyroAngleValues.gyroAngle().getYaw().getDegrees()`, the order of the record values doesn't affect their use